src/averageAges.js

```diff
@@ -36,8 +53,20 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+function hasChild(people, mother) {
+  const children = people.filter(child => child.mother === mother.name);
+
+  return children.length !== 0;
+}
```

Comment:
looks like you implement method some here. Why don't we just use it?

Reply:
we can't use the method some because mothers can have more than 1 child in tests, so I use filter

Reply:
but actually you are right here :)